MAUI/Calendar/date-navigations.md

```diff
@@ -0,0 +1,172 @@
+
+### Forward
+The `Forward` navigation allows you to view the next immediate date of the calendar based on the CalendarViews.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="30"/>
+        </Grid.RowDefinitions>
+        <Calendar:SfCalendar x:Name="Calendar"
+                             View="Month" />
+        <Button x:Name="button" Grid.Row="1" Text="Forward"
+                Clicked="button_Clicked" />
+</Grid>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+private void button_Clicked(object sender, EventArgs e)
+{
+  this.Calendar.Forward();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Backward
+The `Backward` navigation allows you to view the immediate previous date of the calendar based on the CalendarViews.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="30"/>
+        </Grid.RowDefinitions>
+        <Calendar:SfCalendar x:Name="Calendar"
+                             View="Month" />
+        <Button x:Name="button" Grid.Row="1" Text="Backward"
+                Clicked="button_Clicked" />
+</Grid>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+private void button_Clicked(object sender, EventArgs e)
+{
+  this.Calendar.Backward();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Navigation direction
+Views can be navigated by using the `Navigation direction` property either `Vertical` or `Horizontal` directions by setting the `NavigationDirection` property of `Calendar`. The default [NavigationDirection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_NavigationDirection) is `Vertical`.
+
+The following code shows the Navigation direction in `Horizontal`,
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        NavigationDirection="Horizontal">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.NavigationDirection = CalendarNavigationDirection.Horizontal;
+
+{% endhighlight %}
+{% endtabs %}
+
+![horizontal-navigation-direction-in-maui-calendar](images/date-navigation/horizontal-navigation-direction-in-maui-calendar.png)
+
+## Show navigation arrow
+By using the `ShowNavigationArrows` property of the `Calendar` you can navigate to the next or previous views of the calendar without swiping. By default, the value of [ShowNavigationArrows](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarHeaderView.html#Syncfusion_Maui_Calendar_CalendarHeaderView_ShowNavigationArrows) is `true`.
```

Comment:
By using the `ShowNavigationArrows` property of the `Calendar,` you can navigate to the next or previous view of the calendar without swiping. By default, the value of [ShowNavigationArrows](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarHeaderView.html#Syncfusion_Maui_Calendar_CalendarHeaderView_ShowNavigationArrows) is `true.`

Reply:
Content changed